engine/db/table_segment_mvp.cpp

```diff
@@ -306,9 +393,17 @@ Status TableSegmentMVP::Insert(meta::TableSchema& table_schema, Json& records, i
       }
     }
     ++cursor;
+
+  LOOP_END : {}
+    // nothing should be done at the end of block
   }
   record_number_.store(cursor);
-  return Status::OK();
+  auto msg = std::string("loading done. added entries: " +
```

Comment:
auto msg = "Successfully inserted " + std::to_string(new_record_size - skipped_entry) + records."
if (skipped_entry > 0) {
  msg += "Skipped " + std::to_string(skipped_entry) + " records with primary key values that already exist."
}

Reply:
Also, need change WebController to respond the message back to end user

Reply:
messaged improved in e9d07c260c76b57a4e6109815faaada478989bdd
web controller response added in 7be4c52d5e7118fc47c6339c94873080e3fdc95e